overlord/hookstate/ctlcmd/is_connected.go

```diff
@@ -0,0 +1,101 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package ctlcmd
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/overlord/ifacestate"
+)
+
+type isConnectedCommand struct {
+	baseCommand
+
+	Positional struct {
+		PlugOrSlotSpec string `positional-args:"true" positional-arg-name:":<plug|slot>"`
+	} `positional-args:"yes"`
+}
+
+var shortIsConnectedHelp = i18n.G(`The is-connected returns zero if the given plug or slot is connected, and non-zero otherwise`)
```

Comment:
```suggestion
var shortIsConnectedHelp = i18n.G(`return success if the given plug or slot is connected, and failure otherwise`)
```
or,
```suggestion
var shortIsConnectedHelp = i18n.G(`return whether the given plug or slot is connected`)
```

Keep in mind this short help is printed right next to the command itself (in `snapctl -h` for example), so repeating the command just adds length